database.js

```diff
@@ -0,0 +1,27 @@
+async function addNewSoldier(client, newSoldier) {
+  newSoldier.duties = [];
+  const soldierInserted = await client.db('CallOfDuty').collection('Soldiers').insertOne(newSoldier);
```

Comment:
Add vars to `dbName` and `collection` so there will be one single source of truth

Reply:
done